wallet-new/src/Cardano/Wallet/Kernel/Restore.hs

```diff
@@ -379,54 +381,51 @@ restoreWalletHistoryAsync wallet rootId prefilter progress start (tgtHash, tgtSl
     startingPoint <- case start of
         Nothing -> withNode $ getFirstGenesisBlockHash genesisHash
         Just sh -> nextHistoricalHash sh >>= maybe (throwM $ RestorationSuccessorNotFound sh) pure
-    restore genesisHash startingPoint NoTimingData
+    restore genesisHash startingPoint 1000
```

Comment:
Minor one -- might be nice to explicitly have a `where` clause for this "magic number", to clarify its purpose.